doc/MANUAL.adoc

```diff
@@ -94,6 +94,16 @@ Common options
     Clear the entire cache, removing all cached files, but keeping the
     configuration file.
 
+*`--evict-older-than`* _NUM_::
+
+    This is another cleanup mechanism (independent of max_files and max_size).
+    It instructs ccache to perform cleanup based on age of the stored files.
+    The value _NUM_ is duration and means any cache entry older than it from CURRENT_TIME
+    should be deleted. _NUM_ should be an unsigned number with an optional suffix:
+    d(ays)/s(econds). Default being d(ays).
```

Comment:
I would actually prefer to not have a default suffix and require `d` for days. Sounds OK?

Reply:
Okay for me.
Took a while to get back to this :-)